<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="temp_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="hum_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Humidity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_temp">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_hum">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_temp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="list_hum">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graph_temp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graph_hum">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="graph_temp_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </widget>
  <widget class="QLabel" name="graph_hum_label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </widget>
  <widget class="QLabel" name="temp_value">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Current temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="hum_value">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Humidity</string>
   </property>
  </widget>
  <widget class="QLabel" name="temp_time">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="hum_time">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="avg_temp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Avg: </string>
   </property>
  </widget>
  <widget class="QLabel" name="avg_hum">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Avg: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="conversion_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>C/F</string>
   </property>
  </widget>
  <widget class="QLabel" name="alarm_temp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="alarm_hum">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>130</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
